src/Message/PurchaseRequest.php

```diff
@@ -130,7 +130,7 @@ protected function getEndPoint($postdata)
             $timestamp = time();
             $qd['pstn_HMACTimestamp'] = $timestamp;
             $qd['pstn_HMAC'] = $this->getHmac($timestamp, $postdata);
-            $url .= '?'.http_build_query($qd);
+            $url .= '?' . http_build_query($qd);
```

Comment:
Shouldn't be adding spaces here or below - there's nothing in the PSR-2 standard about concat spacing, and omnipay-common uses no spaces around string concatenation as its convention.

Reply:
Heya Leith - sorry about the editor auto-formatting. Fixed now. Cheers.